### ISSUE TITLE
Unsupported platform: arm64-darwin21

### ISSUE THREAD

**Issue by dhniels** [CloudCannon/pagebreak#30]
I get this error when running `gem install jekyll-pagebreak -v '0.6.0' --source 'https://rubygems.org/'`

I am on macOS Monterey 12.1. Does this have to do with the fact I am on an M1 Mac? Any way around this or will I have to wait for the package maintainers to provide support?

Here is the full terminal output:

```
Fetching jekyll-watch-2.2.1.gem
Successfully installed jekyll-watch-2.2.1
Building native extensions. This could take a while...
ERROR:  Error installing jekyll-pagebreak:
	ERROR: Failed to build gem native extension.

    current directory: /Users/dave/.rbenv/versions/3.1.0/lib/ruby/gems/3.1.0/gems/jekyll-pagebreak-0.6.0/ext
/Users/dave/.rbenv/versions/3.1.0/bin/ruby -I /Users/dave/.rbenv/versions/3.1.0/lib/ruby/3.1.0 -r ./siteconf20211225-4010-ygkdrn.rb extconf.rb
Installing Pagebreak
Unsupported platform: arm64-darwin21
Please open an issue at https://github.com/CloudCannon/pagebreak/issues

extconf failed, exit code 1

Gem files will remain installed in /Users/dave/.rbenv/versions/3.1.0/lib/ruby/gems/3.1.0/gems/jekyll-pagebreak-0.6.0 for inspection.
Results logged to /Users/dave/.rbenv/versions/3.1.0/lib/ruby/gems/3.1.0/extensions/arm64-darwin-21/3.1.0/jekyll-pagebreak-0.6.0/gem_make.out
```

**Comment by bglw** [comment]
Hello 👋

I don't have an M1 machine available, but I imagine the Intel build will run on an M1 through Rosetta, so it might be as simple as aliasing that platform flag. 

I just pushed that change on a branch, you can use it directly from git if you update your Gemfile to reference:
```
gem 'pagebreak', git: 'git://github.com/CloudCannon/pagebreak.git', branch: 'm1-support'
```

I'm making this change from my phone so I can't test, but the test suite is passing on #31. I would love to hear whether this works for you, and I can ship a release if it does when I'm next at my laptop.

Cheers!

**Comment by bglw** [comment]
Hi @dhniels — did you get a chance to test whether this worked? Otherwise, I'll track down an M1 to test on before I merge this into main.

**Comment by gliwka** [comment]
I've faced the same issue, the ```m1-support``` branch fixed it for me.

I had to specify the path to the gem spec to get it working with the git repository:
```ruby
  gem 'jekyll-pagebreak', git: 'git://github.com/CloudCannon/pagebreak.git', branch: 'm1-support', glob: 'generator-plugins/jekyll/jekyll-pagebreak.gemspec'
```

**Comment by bglw** [comment]
Thanks @gliwka! This has been released as `0.9.0`, so you should be good to go back to the versioning now.
